src/components/calculator.js

```diff
@@ -1,41 +1,57 @@
-          <button className="btn btn-outline-secondary rounded-0 col" type="button">+/-</button>
-          <button className="btn btn-outline-secondary rounded-0 col" type="button">%</button>
-          <button className="rounded-0 border-secondary operator col" type="button">÷</button>
+          <button className="btn btn-outline-secondary rounded-0 col" onClick={this.handleClick} type="button">AC</button>
+          <button className="btn btn-outline-secondary rounded-0 col" onClick={this.handleClick} type="button">+/-</button>
+          <button className="btn btn-outline-secondary rounded-0 col" onClick={this.handleClick} type="button">%</button>
+          <button className="rounded-0 border-secondary operator col" onClick={this.handleClick} type="button">÷</button>
         </div>
         <div className="row btn-container text-center mx-auto">
-          <button className="btn btn-outline-secondary rounded-0 col" type="button">7</button>
-          <button className="btn btn-outline-secondary rounded-0 col" type="button">8</button>
-          <button className="btn btn-outline-secondary rounded-0 col" type="button">9</button>
-          <button className="rounded-0 border-secondary col operator" type="button">x</button>
+          <button className="btn btn-outline-secondary rounded-0 col" onClick={this.handleClick} type="button">7</button>
+          <button className="btn btn-outline-secondary rounded-0 col" onClick={this.handleClick} type="button">8</button>
+          <button className="btn btn-outline-secondary rounded-0 col" onClick={this.handleClick} type="button">9</button>
+          <button className="rounded-0 border-secondary col operator" onClick={this.handleClick} type="button">x</button>
         </div>
         <div className="row btn-container text-center mx-auto">
-          <button className="btn btn-outline-secondary rounded-0 col" type="button">4</button>
-          <button className="btn btn-outline-secondary rounded-0 col" type="button">5</button>
-          <button className="btn btn-outline-secondary rounded-0 col" type="button">6</button>
-          <button className="rounded-0 border-secondary col operator" type="button">-</button>
+          <button className="btn btn-outline-secondary rounded-0 col" onClick={this.handleClick} type="button">4</button>
+          <button className="btn btn-outline-secondary rounded-0 col" onClick={this.handleClick} type="button">5</button>
+          <button className="btn btn-outline-secondary rounded-0 col" onClick={this.handleClick} type="button">6</button>
+          <button className="rounded-0 border-secondary col operator" onClick={this.handleClick} type="button">-</button>
         </div>
         <div className="row btn-container text-center mx-auto">
-          <button className="btn btn-outline-secondary rounded-0 col" type="button">1</button>
-          <button className="btn btn-outline-secondary rounded-0 col" type="button">2</button>
```

Comment:
[OPTIONAL] When I make an operation I can only see either one operand or the operator instead of all of them. For example, when I attempt 2 + 2 I see a 2 then. when I enter + the 2 disappears  and + shows. I think it would be better if I see the entire 2 + 2